data/AkshayAwate.json

```diff
@@ -4,11 +4,9 @@
   "type": "personal",
   "bio": "Cloud Native Engineer",
   "avatar": "https://github.com/AkshayAwate.png",
-  "links": [
-    {
-      "name": "Follow me on Twitter",
-      "url": "https://twitter.com/AkshayAwate",
-      "icon": "FaTwitter"
-    }
+  "socials": [
+  { "icon": "FaTwitter", "url": "https://twitter.com/AwateAkshay" },
+  { "icon": "FaGithub", "url": "https://github.com/AkshayAwate" },
+  { "icon": "FaLinkedIn", "url": "https://www.linkedin.com/in/akshay-awate%E2%98%B8%EF%B8%8F-ba5a83131/" }
   ]
```

Comment:
Hey @AkshayAwate! This looks good so far, but you have not added any links to your linkfree! The "socials" section shows up just below the avatar as clickable icons.  To complete your profile, you will want to add your links section as well.  To see what I mean, take a look at the docs https://linkfree.eddiehub.io/docs/quickstart and Eddie's profile https://github.com/EddieHubCommunity/LinkFree/blob/main/data/eddiejaoude.json

Let us know if you need any help!